src/style.css

```diff
@@ -0,0 +1,128 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir-bold";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+  font-weight: 900;
+}
+
+/* html,
+body {
+  margin: 0;
+  padding: 0;
+} */
+
+input::-webkit-search-cancel-button {
+  -webkit-appearance: none;
+}
+
+.big-search {
+  display: flex;
+  align-items: center;
+  position: relative;
+  width: 100%;
+  height: 70px;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-top: 20px;
+}
+
+.small-search {
+  display: flex;
```

Comment:
you duplicate to much styles, create common class for two inputs and move common styles to this class